src/java.base/share/classes/java/util/Collections.java

```diff
@@ -5253,6 +5253,18 @@ public boolean removeIf(Predicate<? super E> filter) {
         public int hashCode() {
             return Objects.hashCode(element);
         }
+        public Object[] toArray() {
+            return new Object[] {element};
+        }
+        @SuppressWarnings("unchecked")
+        public <T> T[] toArray(T[] a) {
```

Comment:
Should these have `@Override`?

Reply:
All overriding methods should, yes. Also, there is slight a mismatch between the title and the contents of the PR: it's not only about `ArrayList`.